js/main.js

```diff
@@ -89,74 +128,53 @@
       buildToDoListTable(table);// 表示するノードを組み立て（ループ処理）
       comment.value = '';// 入力テキストを空にする
     };
+    displayTable();
   });
-
-  // 削除ボタンClick時の処理
-  function clickDeleteButton() {
-    const deleteToDoRow = this.parentNode.parentNode; // 削除する行の要素を取得
-    table.innerHTML = defaultTableHtml; // テーブルを初期状態に戻す
-    // deleteToDoRow.remove(); // Clickした要素を削除
-    aryToDo.splice(deleteToDoRow.cells[0].firstChild.data, 1); // 配列の要素を削除（開始位置, 要素数）
-    console.log(aryToDo[0].status); // debug用
-    buildToDoListTable(table);// 表示するノードを組み立て（ループ処理）
-  };
-
-  // 作業中（完了）ボタンClick時の処理
-  function clickWorkingCompleteButton() {
-    const clickToDoRow = this.parentNode.parentNode; // 作業中or完了ボタンを押した行の要素を取得
-    const rowIdOfClickedButton = clickToDoRow.cells[0].firstChild.data; // Clickしたボタンの行のID
-    // console.log(aryToDo[rowIdOfClickedButton].status);  // debug用
-    if (this.className == 'Working') {
-      this.textContent = '完了';
-      this.classList.remove('Working');
-      this.classList.add('Complete');
-      aryToDo[rowIdOfClickedButton].status = "完了"; // 完了にした配列のstatusを作業中⇨完了に変更する
-      // console.log(aryToDo[rowIdOfClickedButton].status); // debug用
-    } else if (this.className == 'Complete') {
-        this.textContent = '作業中';
-        this.classList.remove('Complete');
-        this.classList.add('Working');
-        aryToDo[rowIdOfClickedButton].status = "作業中"; // 完了にした配列のstatusを作業中⇨完了に変更する
-        // console.log(aryToDo[rowIdOfClickedButton].status); // debug用
-    };
-  };
-
+  
+  // Radioボタン（すべて）CheckにChange時の操作
   const radioButtonAll = document.getElementById('all');
+  radioButtonAll.addEventListener('change', function() {

```

Comment:
displayTable()はグローバルな関数ですので、このように修正しましょう。
```suggestion
  radioButtonAll.addEventListener('change', displayTable);

```